src/vfb_query_builder/test/query_roller_tests.py

```diff
@@ -52,6 +52,53 @@ def get_edits_string(old, new):
                             break
         print('New vfb.xmi')
         print(''.join(xmi))
-                            
+
+        queries = json.loads(term_info_export(escape='json'))
+        # extract out the cypher queries from the VFBconnect master branch
+        ecore=urllib.request.urlopen("https://github.com/VirtualFlyBrain/VFB_connect/raw/master/src/vfb_connect/resources/VFB_TermInfo_queries.json")
+        model = []
+        xmi = []
```

Comment:
do we need both xmi and model? Will this do the same job:
```
        for line in ecore:
            xmi.append(line.decode('utf-8'))
        for query in queries:
            print(query)
            print(newquery(queries[query]))
            for index, line in enumerate(xmi):
                if query in line:
                    xmi[index] = queries[query]
                    # break
```

Reply:
note: seems this won't work if brace brackets are in the same line with the queries as in VFB_results_single_input.json